doc/user_guide/install.rst

```diff
@@ -285,32 +285,31 @@ To get the development version from our git repository you need to install `git
 
 .. Warning::
 
-    When running hyperspy from a development version, it can happen that the
-    dependency requirement changes in which you will need to keep this
-    this requirement up to date (check dependency requirement in ``setup.py``)
-    or run again the installation in development mode using ``pip`` as explained
-    below.
+    When running hyperspy from a development version, it can happen that
+    dependencies change when pulling new commits / syncing the project. You 
+    will then need to manually update dependencies in the environment 
+    (check `pyproject.toml <https://github.com/hyperspy/hyperspy/blob/RELEASE_next_minor/pyproject.toml>`__) 
+    or run again the development installation procedure, as explained below.
 
 Installation in a Anaconda/Miniconda distribution
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Optionally, create an environment to separate your hyperspy installation from
 other anaconda environments (`read more about environments here
-<https://conda.io/projects/conda/en/latest/user-guide/concepts/environments.html>`_):
+<https://conda.io/projects/conda/en/latest/user-guide/concepts/environments.html>`_).
+At this point, it is helpful to let conda pick compatible ``python`` and 
+``hyperspy-dev`` versions. ``hyperspy-dev`` contains development 
+dependencies required for testing and building documentation. 
 
 .. code-block:: bash
 
-    $ conda create -n hspy_dev python # create an empty environment with latest python
+    $ conda create -n hspy_dev python hyperspy-dev -c conda-forge # create an env with compatible hyperspy-dev and python versions
     $ conda activate hspy_dev # activate environment
 
-Install the runtime and development dependencies requirements using conda:
+Install the runtime requirements using conda:
 
 .. code-block:: bash
 
     $ conda install hyperspy-base -c conda-forge --only-deps # install hyperspy dependencies
```

Comment:
Not sure, if this line is still required then?

Reply:
I think so, otherwise you wouldn't get rosettasciio, scipy, etc. no? 

Since I don't know how to consult what's in hyperspy-dev I can't tell for sure (also not an expert), but I think the dependencies of hyperspy-base are not in there